### PR TITLE
Util(validate, color) 등록했습니다.

### PR DESCRIPTION
**Pull request by chepchep2**
Util, Color 파일 작성했습니다.

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/auth_color.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+
+class AppColor {
+  static const whiteColor = Color(0xffffffff);
+  static const greyColor = Color(0xFFF2F4F6);
+  static const blackColor = Color(0xFF000000);
+  static const blackColor10 = Color(0x10000000);
```
Comment (flowkater) [inline lib/common/auth_color.dart:7]: 피그마 네이밍 그대로 가져온거죠?

File: lib/common/auth_color.dart
Comment (flowkater) [inline lib/common/auth_color.dart:1]: 파일 이름이 app_color.dart 로 바뀌어야할 것 같습니다.

**Review by flowkater — CHANGES_REQUESTED** [review]
코멘트 확인 부탁해요

File: lib/common/auth_color.dart (hunk 1 above)
Comment (chepchep2) [reply lib/common/auth_color.dart:7]: 이 스택은 피그마 네이밍 그대로 가져온 것은 아니고 최신 스택에 피그마 네이밍을 사용했습니다.

File: lib/common/auth_color.dart
Comment (chepchep2) [reply lib/common/auth_color.dart:1]: 넵 수정하겠습니다.

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18).
* **Mar 17, 8:12 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18).

**Pull request merged**